middleware/kubernetes/controller.go

```diff
@@ -67,7 +70,7 @@ type dnsControl struct {
 }
 
 // newDNSController creates a controller for CoreDNS.
-func newdnsController(kubeClient *kubernetes.Clientset, resyncPeriod time.Duration, lselector *labels.Selector, initPodCache bool) *dnsControl {
+func newdnsController(kubeClient *kubernetes.Clientset, resyncPeriod time.Duration, lselector *labels.Selector, initPodCache bool, initNodeCache bool) *dnsControl {
```

Comment:
which params do you think should go in it?
just the two bools? or also the resyncPeriod and label selector?